constructor/main.py

```diff
@@ -187,26 +187,27 @@ def main_build(dir_path, output_dir='.', platform=cc_platform,
             else:
                 env_config[config_key] = value
 
-    if exe_type is None or exe_version is None:
-        logger.warning(
-            "Could not identify conda-standalone / micromamba version. "
-            "Will assume it is compatible with shortcuts."
+    # Installers will provide shortcut options and features only if the user
+    # didn't opt-out by setting every `menu_packages` item to an empty list
```

Comment:
```suggestion
    # Installers will disable shortcut options and features only if the user
    # opted-out by setting every `menu_packages` item to an empty list
```

I know this wasn't you, but I think this describes what's going on here better since you actively have to opt out instead of opting in.